.gitignore

```diff
@@ -58,3 +58,4 @@ gemfiles/*.lock
 /spec/dummy/app/assets/javascripts/routes.js
 /spec/dummy/logs
 /spec/dummy/tmp
+.idea
```

Comment:
you should use local gitignore instead of addition your things to the repo git ignore. It can go forever otherwise....